depends/Makefile

```diff
@@ -25,6 +25,7 @@ lib/libpfft_omp.a:
 
 clean:
 	rm -rf $(LIBRARIES)
+	rm -rf tmp-pfft-*/double
+	rm -rf tmp-pfft-*/single
```

Comment:
Sometimes I find it more frequent to clean fast libraries but keep the fft library.

Perhaps adding a clean-all target that cleans all and keep clean as clean-fast?

Reply:
How about 
clean-all: clean clean-fft
clean-fft: rm -rf tmp-pfft-*/double
 That preserves back-compat because the meaning of clean does not change

Reply:
What about:

clean: clean-fast clean-fft

back-compat is less important than getting it right in the long term.

Reply:
Ok, I'll do that